gce_image_import_export_tests/test_suites/import/image_import_test_suite.go

```diff
@@ -18,12 +18,13 @@ package importtestsuites
 import (
 	"context"
 	"fmt"
+	"github.com/GoogleCloudPlatform/compute-image-tools/gce_image_import_export_tests/compute"
```

Comment:
This should go to the external imports group below

Reply:
I hate this - always moved by IDE automatically